src/vsr/journal.zig

```diff
@@ -1537,6 +1592,7 @@ pub fn Journal(comptime Replica: type, comptime Storage: type) type {
                 assert(!self.prepare_inhabited[slot.index]);
                 self.prepare_inhabited[slot.index] = true;
                 self.prepare_checksums[slot.index] = message.header.checksum;
+                self.headers_redundant[slot.index] = message.header.*;
```

Comment:
Have we audited all the places where we assign to `prepare_checksums` to check that we do the same for `headers_redundant`? Is there way we can enforce this through a helper?

Reply:
Great, agreed.